Automattic/newspack-newsletters
# Append utm_medium param to all links in newsletter

### All Submissions:






### Changes proposed in this Pull Request:



For targeting visitors coming in from a Newsletter, all links in the newsletter will have a ~`utm_source`~ `utm_medium=email` parameter appended. ~The value of this parameter is `Newsletter` by default, and can be changed in the plugin settings.~

### How to test the changes in this Pull Request:

1. Create a newsletter with some links in the content. 
2. Send the campaign, observe that the links (after MC redirection) have a ~`utm_source=Newsletter`~ `utm_medium=email` parameter
3. ~Change the value of the parameter in plugin settings~
4. ~Send another newsletter with links – observe the parameter is updated~

### Other information:
